### PR TITLE
CI: auto label PR's

### PR DESCRIPTION
**Pull request by jnewb1**
i usually forget

### CODE REVIEW

**Comment by jnewb1** [comment]
might have some annoying false positives, ex: if you are doing a refactor and changed a bunch of files, but probably useful overall

**Review by adeebshihadeh — CHANGES_REQUESTED** [review]
Looks like this will have a bunch of FPs that are annoying (e.g. all UI PRs that touch text will now be labelled with "multilang"). Let's start with some simple lables that won't have too many FPs and expand from there.

**Comment by jnewb1** [comment]
see any problems with enabling it for cars and CI? (other than if you refactor all the cars at once you will get every car's label)

**Comment by adeebshihadeh** [comment]
I can still see a ton of FPs with those. Can we do something like "if only selfdrive/car/toyota is touched, label as toyota"? If not, perhaps there's a different action out there with more options?

**Comment by jnewb1** [comment]
looks like "all" is what we want. If "all" the files changed are in /toyota, apply the toyota label

**Comment by adeebshihadeh** [comment]
Nice, then we should be able to do a bunch with basically no FPs.

File: .github/labeler.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  - all: ['selfdrive/car/volkswagen/*']
+
+simulation:
+  - all: ['tools/sim/**']
+ui:
+  - all: ['selfdrive/ui/**']
+tools: 
+  - all: ['tools/**']
+
+#multilanguage: selfdrive/ui/translations/**
```
Comment (adeebshihadeh) [inline .github/labeler.yaml:39]: should be able to do this one too

**Review by adeebshihadeh — APPROVED** [review]

File: .github/labeler.yaml (hunk 1 above)
Comment (jnewb1) [reply .github/labeler.yaml:39]: probably all except main_en?

Comment (adeebshihadeh) [reply .github/labeler.yaml:39]: can't we just do all in translations/**?

Comment (jnewb1) [reply .github/labeler.yaml:39]: ah, yes nvm

**Comment by sshane** [comment]
This seems to be removing labels incorrectly on my PR (and not adding labels when it should): https://github.com/commaai/openpilot/pull/29880

**Comment by jnewb1** [comment]
> This seems to be removing labels incorrectly on my PR (and not adding labels when it should): #29880

It's set to only add labels when it's 100% sure its not a false positive, and apparently the sync-labels doesn't work for manually added labels? might have to revert this

**Comment by sshane** [comment]
Well I simply modified a files in the selfdrive/car folder which should match multiple labels, yet it doesn't add them. That seems like a separate bug, no?

**Comment by jnewb1** [comment]
> Well I simply modified files in the selfdrive/car folder which should match multiple labels, yet it doesn't add them. That seems like a separate bug, no?

I believe it includes the opendbc

**Comment by sshane** [comment]
Ah, so the full set of changed files needs to be in that label's list, or it doesn't add it? So if you modify `/tools` and `selfdrive/car/**`, it won't add any labels?

**Comment by jnewb1** [comment]
> Ah, so the full set of changed files needs to be in that label's list, or it doesn't add it? So if you modify `/tools` and a directory without a label, it won't add any labels?

Yes, adeeb didn’t want any false positives, example of you change all the car brands for a refactor you would get all the car labels

**Pull request merged**